common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -175,6 +175,8 @@
   "feature.wynntils.questInfoOverlay.name": "Quest Status Overlay Feature",
   "feature.wynntils.questInfoOverlay.overlay.questInfo.description": "Display the currently tracked quest's info",
   "feature.wynntils.questInfoOverlay.overlay.questInfo.name": "Quest Info Tracker",
+  "feature.wynntils.questInfoOverlay.overlay.testQuestName": "Test quest",
```

Comment:
Btw this should be `feature.wynntils.questInfoOverlay.overlay.questInfo.testQuestName` to match our format.